app/views/partials/_articles.html.erb

```diff
@@ -0,0 +1,69 @@
+<% total_articles = total_articles ||= 6 %>
+
+<% @article_types = ArticleType.all %>
+
+<% news_page = @cms_site.pages.find_by_slug('news-and-documents') %>
+<% unless news_page.to_s.strip.empty? %> 
```

Comment:
I think you're calling `.to_s` on the object, so you'll end up with the id of the object, which will always be present